extensions/resource-deployment/src/ui/validation/validations.ts

```diff
@@ -161,8 +203,7 @@ export async function validateInputBoxComponent(component: azdata.InputBoxCompon
 	for (const validation of validations) {
 		const result = await validation.validate();
 		if (!result.valid) {
-			component.updateProperty('validationErrorMessage', result.message);
-			return false;
+			return false; //bail out on first failure, remaining validations are processed after this one has been fixed by the user.
```

Comment:
I assume you're updating the message somewhere else?

Reply:
I am not quite sure what you mean. The component.updateProperty call updates the message value and the fact the validation code here returns false signals to the core that the corresponding field has an invalid value.

Reply:
I'm saying that the code used to have this `component.updateProperty('validationErrorMessage', result.message);` but doesn't anymore. Is that functionality not needed anymore?

Reply:
yes, the equivalent is done now on the onValidation callback that is registered when the Validation object is created. https://github.com/microsoft/azuredatastudio/blob/90a2b2fe3c89661f5a7a6502dffbb94c34487d64/extensions/resource-deployment/src/ui/modelViewUtils.ts#L422-L422